plugins/Toolbox/resources/qml/ToolboxProgressButton.qml

```diff
@@ -5,8 +5,9 @@ import QtQuick 2.2
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import UM 1.1 as UM
+import Cura 1.0 as Cura
 
-
+// TODO; This is in quite some need for refactoring.
```

Comment:
I guess this line can be removed? You did a lot of refactoring for this file. @nallath